s-pipes-core/src/main/java/cz/cvut/spipes/engine/VariablesBinding.java

```diff
@@ -179,9 +179,24 @@ public void load(final InputStream is, final String lang) throws IOException {
 
     @Override
     public String toString() {
-        return binding.asMap().toString();
+        return binding.asMap()
+                .entrySet()
+                .stream()
+                .map(entry -> entry.getKey() + "=" + toString(entry.getValue()))
+                .collect(Collectors.joining(", ", "{", "}"));
```

Comment:
Not sure but isn't it same as:
```suggestion
        return binding.asMap()
                    .entrySet()
                    .stream()
                    .collect(Collectors.toMap(
                         Map.Entry::getKey, 
                         e -> toString(e.getValue())
                     ))
                     .toString();
```

Reply:
done